actionview/test/template/dependency_tracker_test.rb

```diff
@@ -195,11 +205,11 @@ def test_finds_dependencies_with_extra_spaces
 
   def test_dependencies_with_interpolation
     template = FakeTemplate.new(%q{
-      <%# render "double/#{quote}" %>
-      <%# render 'single/#{quote}' %>
+      <%= render "double/#{quote}" %>
+      <%= render 'single/#{quote}' %>
     }, :erb)
     tracker = make_tracker("interpolation/_string", template)
 
-    assert_equal ["single/\#{quote}"], tracker.dependencies
+    assert_equal ["double/", "single/\#{quote}"], tracker.dependencies
```

Comment:
This test shouldn't change expectations. I need to resolve double quoted strings with interpolation in a different way.